### PR TITLE
CASEFLOW-3730 Creation of the "Convert hearing to virtual task action"

### PR DESCRIPTION
**Pull request by j-n-t-l**
Resolves #[CASEFLOW-3730](https://vajira.max.gov/browse/CASEFLOW-3730)
Description
As a VSO user I need the ability to access the convert-to-virtual form so that I am able to opt the Veteran/Appellant I represent into a Virtual hearing.

Acceptance Criteria

- [ ] VSO user has the following task action available for cases they represent: Convert hearing to virtual
- [ ] When the "Convert hearing to virtual" task action is selected the user is directed to the convert-to-virtual form

### CODE REVIEW

**Review by ksross-va — CHANGES_REQUESTED** [review]
spec/models/appellant_substitution_spec.rb needs to be updated to fix some [failing tests](https://app.circleci.com/pipelines/github/department-of-veterans-affairs/caseflow/51727/workflows/8ddd7259-a04e-482d-8c5a-aaccf0d20262/jobs/207506). This may just be a matter of adding ChangeHearingRequestTypeTask as an expected_task_type on line 186.

spec/workflows/initial_tasks_factory_spec.rb should also be updated to include some new test cases making sure ChangeHearingRequestTypeTask gets created when it should (and doesn't generate duplicates)

File: app/workflows/initial_tasks_factory.rb [hunk 1]
```diff
@@ -36,6 +36,9 @@ def create_vso_tracking_tasks
       if TrackVeteranTask.where(appeal: @appeal, assigned_to: rep).empty?
         TrackVeteranTask.create!(appeal: @appeal, parent: @root_task, assigned_to: rep)
       end
+      if ChangeHearingRequestTypeTask.where(appeal: @appeal, assigned_to: rep).empty?
```
Comment (ksross-va) [inline app/workflows/initial_tasks_factory.rb:39]: This conditional logic needs to be updated to only create the ChangeHearingRequestTypeTask if `@appeal.docket_type == "hearing"` (See [AC 1.1](https://vajira.max.gov/browse/CASEFLOW-1520))

```suggestion
      if ChangeHearingRequestTypeTask.where(appeal: @appeal, assigned_to: rep).empty? &&
         @appeal.docket_type == "hearing"
```

**Review by ksross-va — CHANGES_REQUESTED** [review]

**Review by ksross-va — APPROVED** [review]
